packages/components/molecules/f-rating/src/components/_tests/Rating.test.js

```diff
@@ -75,38 +90,44 @@ describe('Rating', () => {
                 });
 
                 // Act & Assert
-                expect(wrapper.vm.getRatingDescription).toBeDefined();
+                expect(wrapper.vm.setRatingVariant).toBeDefined();
             });
 
             describe('when invoked', () => {
-                it('should return a singular description if the rating is less than 2', () => {
-                    // Arrange
-                    propsData = {
-                        starRating: 1
-                    };
-                    wrapper = shallowMount(VRating, {
-                        propsData,
-                        localVue,
-                        i18n
-                    });
+                describe('and prop `isSingleStarVariant` is truthy', () => {
+                    it('should return the single star rating variant: `rating-single-star`', () => {
+                        // Arrange
+                        propsData = {
+                            starRating: 2,
+                            isSingleStarVariant: true
+                        };
+                        wrapper = shallowMount(VRating, {
+                            propsData,
+                            localVue,
+                            i18n
+                        });
 
-                    // Act & Assert
-                    expect(wrapper.vm.getRatingDescription).toMatchSnapshot();
+                        // Act & Assert
+                        expect(wrapper.vm.setRatingVariant).toBe('rating-single-star');
```

Comment:
You can also check for the actual component by importing it and doing something like:
```javascript
expect(wrapper.findComponent(RatingSingleStar).exists()).toBe(true);
```

Reply:
Nice